bee.php

```diff
@@ -44,10 +44,11 @@
 function bee_error_handler($errno, $message, $filename, $line, $context = NULL) {
   if (error_reporting() > 0) {
     // Core uses the @ error operator in url_stat() to suppress the warning for
-    // non-existent files. But that doesn't work for us in PHP 8+.
+    // non-existent files. But that doesn't work for us, as since PHP 8+ the
```

Comment:
Wasn't sure about this, so checked with my wife, who is a proofreader, and she suggested this was the clearest way.
```suggestion
    // non-existent files. But that doesn't work for us, as, since PHP 8+, the
```

Reply:
I don't know much about English comma rules (heck, never mastered the German ones), but that looks a bit odd: `us, as, `

Reply:
I'm not sure, that's why I asked my wife.  Trying to find a suitable phrase that works is tricky.  I went to look at what the documentation says, and am confused as to why we're getting this as [`stat()`](https://www.php.net/manual/en/function.stat.php) is supposed to produce `E_WARNING` on failure which should still be suppressed but isn't.  What about instead of `But that doesn't work for us, as since PHP 8+ the error_reporting value raises automatically with "@".` we do:
`However, since PHP 8.0, certain errors are no longer suppressed which causes unnecessary error messages to appear in some 'bee' functions.`

Reply:
> However, since PHP 8.0, certain errors are no longer suppressed which causes unnecessary error messages to appear in some 'bee' functions.

That sounds a lot better. :+1: One tiny suggestions, maybe `in some Bee commands` - or do you prefer "function"?

Reply:
They are referred to as command in the help and wiki.
https://github.com/backdrop-contrib/bee/wiki/Usage

```php
'help' => array(
      'description' => bt("Provide help and examples for 'bee' and its commands."),
```